File: source/aws/cs/lambda-docker-websocket/UniRpc/WebsocketServiceBase.cs
using System.Threading.Tasks;

namespace UniRpc
{
    public abstract class WebsocketServiceBase
    {
        public WebsocketService __websocketService { get; set; }
        public string __reflection { get; set; }
        public bool __isGeneric { get; set; }
        public string[] __genericArguments { get; set; }
        public abstract void __outgoing(BaseMessage message);
        public string __user { get; set; }
        public string[] __groups { get; set; }
        public abstract Task<BaseMessage> __invoke(BaseMessage message);
    }
}

File: source/aws/ts/lambda-docker-websocket/UniRpc/WebsocketServiceBase.ts
import { BaseMessage } from './BaseMessage';
import { WebsocketService } from './WebsocketService';

export abstract class WebsocketServiceBase
{
    __websocketService: WebsocketService;
    __reflection: string;
    __isGeneric: boolean;
    __genericArguments: string[];
    abstract __outgoing(message: BaseMessage): void;
    __user: string;
    __group: string;
    abstract __invoke(message: BaseMessage): Promise<BaseMessage>;
}